Week-4/session-11/Prototype and Object/object.js

```diff
@@ -0,0 +1,34 @@
+const person = {
+	_name: 'John Doe',
+	_age: 30,
+	_email: 'john.doe@example.com',
+	get name() {
+		return this._name;
+	},
+	get email() {
+		return this._email;
+	},
+	set age(newAge) {
+		this._age = newAge;
+	},
+	getAge() {
+		return this._age;
+	},
+	setAge(newAge) {
+		if (typeof newAge === 'number' && newAge >= 0) {
+			this._age = newAge;
+		} else {
+			console.log('Invalid age value');
+		}
+	},
+};
+
+console.log('Name:', person.name);
+console.log('Email:', person.email);
+
+person.age = 35;
+console.log('Age:', person.getAge());
```

Comment:
image.png

this is accessible , but as per ask it should be private

Reply:
<img width="596" alt="image" src="https://github.com/pesto-students/p14-harsh-srivastava-deepakk5893/assets/127043221/db23d753-5170-482e-87e3-7041a874b8c7">